joomla/joomla-cms
# [4.0] More state => stage renaming

### Summary of Changes
Renaming the filter options to fit the new naming.


### Testing Instructions
Go to the article (or featured article) manager and check out the filter options
